lib/evmmax/evmmax.cpp

```diff
@@ -0,0 +1,114 @@
+// evmone: Fast Ethereum Virtual Machine implementation
+// Copyright 2023 The evmone Authors.
+// SPDX-License-Identifier: Apache-2.0
+
+#include <evmmax/evmmax.hpp>
+
+using namespace intx;
+
+namespace evmmax
+{
+namespace
+{
+inline constexpr uint64_t mul_inv64(uint64_t base) noexcept
+{
+    uint64_t result = 1;
+    for (auto _ = 0; _ < 64; ++_)
```

Comment:
```suggestion
    for (auto i = 0; i < 64; ++i)
```

Reply:
Changed. But for the reference, C++ wants to standardize `_` as the "discard" variable name (can't find the paper right now).